tests/endpoint-test.js

```diff
@@ -4528,6 +4528,49 @@ describe('Endpoint', function() {
 			});
 		});
 
+		describe('getFilesUnderRetentionForAssignment()', function() {
```

Comment:
can you also add equivalent tests to https://github.com/box/box-node-sdk/blob/main/tests/lib/managers/retention-policies-test.js#L478?